main.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+let list = document.querySelector('.tree');
+
+list.addEventListener('click', event => {
+if(event.target.parentNode.childNodes.length > 1) {
+    if (event.target.parentNode.childNodes[1].style.display === 'none') {
```

Comment:
do not modify styles directly, toggle classes or use `hidden` attribute
create variable if you need to access some object property more then once